sound/soc/sof/Kconfig

```diff
@@ -188,6 +188,14 @@ config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
 	  Say Y if you want to enable IPC flood test.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DEBUG_IPC_MSG_INJECTOR
+	bool "SOF enable IPC message injector"
+	help
+	  This option enables the IPC message injector which can be used to send
+	  crafted IPC messages to the DSP to test it's robustness.
+	  Say Y if you want to enable IPC flood test.
```

Comment:
copy-paste? this doesn't look related to the IPC flood test?

Reply:
Ah, yes.